docs/linux/setup_ubuntu-host_android-device_arm32-kernel.md

```diff
@@ -0,0 +1,84 @@
+# Setup: Ubuntu host, arm32 kernel on an Android device
+
+This document will detail the steps involved in setting up a syzkaller instance fuzzing an ARM32 linux kernel on an Android (or Android Things) device. This is a work-in-progress at this time and being provided to spur further development. Some features of syzkaller may not yet work on ARM32. For example, not all debugging and test coverage features are available in the Linux kernel for ARM32, limiting the efficacy of syskaller in finding bugs fast. These instructions help set up syzkaller to be a basic fuzzer that does not rely on test coverage data from the kernel. 
+
+## Install Android and Linux kernel on an ARM32 device
+
+Follow the instructions for the ARM32 board to install Android or 
+Android Things and make sure the device boots properly.
+
+Set up the adb bridge so that adb and fastboot work.
+
+Setup a serial port, following the instructions for your board so that you can monitor any messages from the kernel.
+
+These were tested on an NXP Pico-Pi-IMX7D following the instructions [here] (
```

Comment:
No space between `]` and `(`.
Otherwise it is not parsed as link by github.
You can preview how it will look by clinking "View" in top-right corner of this file.